radish34/api/package.json

```diff
@@ -17,7 +17,7 @@
   "scripts": {
     "build": "babel src -d dist",
     "start": "npm run build && node ./dist/index.js",
-    "dev": "npm run build && nodemon --exec babel-node ./src/index.js",
+    "dev": "npm run start && nodemon --exec babel-node ./src/index.js",
```

Comment:
I think we should leave this as:
```
"dev": "npm run build && nodemon --exec babel-node ./src/index.js",
```
I don't think there is a need to start a `node` and a `nodemon` process. Should be one or the other